### DISCUSSION TITLE
Welcome to HA-Battery-Notes Discussions!

### DISCUSSION THREAD

**Discussion by andrew-codechimp** [andrew-codechimp/HA-Battery-Notes#418] [Announcements]
## 👋 Welcome!
  Due to the phenomenal attention my little integration has got I'm opening discussions for when an issue doesn't seem the right place.

  I want to use the ideas section to capture great enhancement ideas and gather feedback on how best to make them work for everyone.  I'll be moving new feature issues into here so they aren't lost but don't clog up the issues log if I'm not going to get round to them for some time.

  I hope that you use the discussions are to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you're using Battery Notes with this community.

**Comment by andrew-codechimp** [comment]
So I guess I should introduce myself as the creator of this integration.

I've been a developer since the late 80's but don't code as part of my day job for about 10 years now so coding is more of a sideline hobby.  I've touch most languages/operating systems VB, Perl, Java, Swift, Windows, Android, IOS to name a few and most recently Python.  Certainly no expert but I have just enough skill to get things working fairly robustly if not elegantly.

I've been a Home Assistant user for about 8 years now, I have the usual lights, buttons, motions sensors, relays and power monitoring but always on the lookout on what I can integrate/automate next.

I live in the UK and tend to be more active on my pet projects first things in the morning and at weekends, I try not to look at GitHub in an evening as it keeps me awake thinking about ideas or problems 😄

**Reply by hellcry37** [reply]
I'd like to congratulate you for your awesome work and dedication to this integration, I'm enjoying it very much, thank you!

**Reply by andrew-codechimp** [reply]
Thanks and glad you're getting use out of it. It's been interesting learning how people want to monitor batteries and has even improved my own management of them.

**Reply by hellcry37** [reply]
I've made a reddit post about it hope it caches up:
https://www.reddit.com/r/homeassistant/comments/1agzuev/battery_management/

Again very nice job and thank you!

**Comment by SteveButt** [comment]
Hi Andrew, Great work, just installed it today. The one thing missing for me is that I use the blink integration and it would be great to get warning when those batteries are low. Is that a problem with the blink integration or is it something that could be picked up by this using the voltage value ? Looking at the diagnostics it seems the cameras publish 3 battery related values 
"battery": "ok",
"battery_level": null,
"battery_voltage": 146,

Battery Level doesnt look particularly helpfull but maybe that a bug in the blink integration ?

If battery_level had a value would your project pick it up ?

**Reply by andrew-codechimp** [reply]
Hi, glad you are enjoying it so far.

Blink seems an odd integration, battery should be a percentage type if it was sticking with HA conventions so the Battery+ won't work.
You can get the battery low binary sensor within Battery Notes working by adding a template based on the battery voltage, go in and configure the battery note for this device adding something like the third example on [this page ](https://andrew-codechimp.github.io/HA-Battery-Notes/#battery-low-template) Change the < 1 to something more relevant when you see the lower voltage on your sensor, if it's starting off at 146 full maybe 50?  The float(5) is the default value if the device isn't returning a value, which again you might want to tweak.

**Comment by misc-brabs** [comment]
Hi Sir,
First and foremost, kudos for making available one of the most useful add-on of all.

The idea of adding both entities that are linked to the battery powered device BUT also a battery+ entity with rich attributes is just great.
This way we can cover most use cases inclusive of using these attributes from battery+ with auto-entities card for display and/or filtering
(as often, attributes is the only way to do so to limit repeat and messy copy/paste). So elegant!

Also, your add-on is quasi flawless and apply smoothly to my collection of 50+ battery powered devices leveraging on the ever increasing library. Fact that you are doing this a hobby makes it even more impressive.

Just great, so a great thank again to you for contributing so much to the home assistant expanding community.

Best

Axel
ps: been a HA user for 1y+ and learning everyday...

**Reply by andrew-codechimp** [reply]
Thanks so much for the positive comments, the community that has grown around this project has been great, I've learnt so much on how people want to maintain their batteries.  HA user for about 8 years and still learning!
